<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegistrWindoww</class>
 <widget class="QWidget" name="RegistrWindoww">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>438</width>
    <height>486</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>438</width>
    <height>486</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>438</width>
    <height>486</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:  qlineargradient( spread:pad,  x1:0, y1:0, x2:1, y2:0,  stop:0 rgba(105, 57, 153, 255),  stop:1 rgba(165, 105, 189, 255));  border: 1px solid #5D3B9B;  border-radius: 5px;
font-family: Everlasting;</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>30</y>
     <width>301</width>
     <height>434</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">backgroud-color: rgba(255, 255, 255, 30);
border: 1px solid rgba(255, 255, 255, 40);
border-radius: 7px;</string>
   </property>
   <widget class="QLineEdit" name="lineEdit_4">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>170</y>
      <width>161</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;
padding-left: 10px;</string>
    </property>
    <property name="maxLength">
     <number>12</number>
    </property>
    <property name="placeholderText">
     <string>9500852641</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Return_button_Clicked">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>383</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
color: white;
background-color: rgba(255,255,255,30);
border: 1px solid rgba(255,255,255,40);
border-radius: 12px;
} 
QPushButton:hover{
background-color: rgba(255,255,255,40);
}

QPushButton:pressed{
background-color: rgba(255,255,255,70);
}</string>
    </property>
    <property name="text">
     <string>Вернуться</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>110</y>
      <width>161</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;
padding-left: 10px;</string>
    </property>
    <property name="maxLength">
     <number>30</number>
    </property>
    <property name="placeholderText">
     <string>Иван</string>
    </property>
   </widget>
   <widget class="QPushButton" name="reg_button_Clicked">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>330</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
color: white;
background-color: rgba(255,255,255,30);
border: 1px solid rgba(255,255,255,40);
border-radius: 12px;
font-size: 10pt;
} 
QPushButton:hover{
background-color: rgba(255,255,255,40);
}

QPushButton:pressed{
background-color: rgba(255,255,255,70);
}</string>
    </property>
    <property name="text">
     <string>Зарегистрироваться</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>70</y>
      <width>161</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;
padding-left: 10px;</string>
    </property>
    <property name="maxLength">
     <number>30</number>
    </property>
    <property name="placeholderText">
     <string>Иванов</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_3">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>140</y>
      <width>161</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;
padding-left: 10px;</string>
    </property>
    <property name="maxLength">
     <number>30</number>
    </property>
    <property name="placeholderText">
     <string>Иванович</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>20</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;
font-weight: bold;
border: none;
font-size: 12pt;
background-color: none;</string>
    </property>
    <property name="text">
     <string>Регистрация</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_5">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>200</y>
      <width>161</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;
padding-left: 10px;</string>
    </property>
    <property name="maxLength">
     <number>50</number>
    </property>
    <property name="placeholderText">
     <string>Пушкина 21</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_6">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>240</y>
      <width>161</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;
padding-left: 10px;</string>
    </property>
    <property name="maxLength">
     <number>50</number>
    </property>
    <property name="placeholderText">
     <string>login</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_7">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>280</y>
      <width>161</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;
padding-left: 10px;</string>
    </property>
    <property name="maxLength">
     <number>50</number>
    </property>
    <property name="placeholderText">
     <string>password</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;
font-weight: bold;
font-size: 9pt;
border: none;
background-color: none;</string>
    </property>
    <property name="text">
     <string>Фамилия</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;
font-weight: bold;
font-size: 9pt;
border: none;
background-color: none;</string>
    </property>
    <property name="text">
     <string>Имя</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;
font-weight: bold;
font-size: 9pt;
border: none;
background-color: none;</string>
    </property>
    <property name="text">
     <string>Отчество</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;
font-weight: bold;
font-size: 9pt;
border: none;
background-color: none;</string>
    </property>
    <property name="text">
     <string>Телефон</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;
font-weight: bold;
font-size: 9pt;
border: none;
background-color: none;</string>
    </property>
    <property name="text">
     <string>Адрес </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;
font-weight: bold;
font-size: 9pt;
border: none;
background-color: none;</string>
    </property>
    <property name="text">
     <string>Логин</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;
font-weight: bold;
font-size: 9pt;
border: none;
background-color: none;</string>
    </property>
    <property name="text">
     <string>Пароль</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
